jwk/jwk_internal_test.go

```diff
@@ -191,3 +191,56 @@ func TestIterator(t *testing.T) {
 		})
 	}
 }
+
+func TestIsPrivate(t *testing.T) {
```

Comment:
This internal test thing is going to go away, so I'd like to refrain from adding tests in this file. Can you please test through the public API. Yeah, I know it's a bit of a pain, but I'm sure you can find source for all of the key types throughout the test